dask/dataframe/core.py

```diff
@@ -5951,7 +6020,10 @@ def handle_out(out, result):
         msg = (
             "The out parameter is not fully supported."
             " Received type %s, expected %s "
-            % (typename(type(out)), typename(type(result)))
+            % (
+                typename(type(out)),
+                typename(type(result)),
+            )
         )
```

Comment:
Is this change necessary? If you want to keep it, you might as well switch it to `f`-string.

Reply:
I did not change that actually, maybe it was a consequence of automatic formatting?